### PR TITLE
fixed build on ubuntu 16.10 (boost 1.61 and gcc6)

### PR DESCRIPTION
**Pull request by planthaber**

### CODE REVIEW

File: src/TypelibConfiguration.cpp [hunk 1]
```diff
@@ -84,7 +84,7 @@ std::shared_ptr< ConfigValue > TypelibConfiguration::getFromNumeric(Typelib::Val
                     break;
                 default:
                     std::cout << "Error, got integer of unexpected size " << numeric.getSize() << std::endl;
-                    return false;
+                    return std::shared_ptr< ConfigValue >(new SimpleConfigValue("Nothing"));
```
Comment (jmachowinski) [inline src/TypelibConfiguration.cpp]: Just Throw here...

File: src/TypelibConfiguration.cpp [hunk 2]
```diff
@@ -106,7 +106,7 @@ std::shared_ptr< ConfigValue > TypelibConfiguration::getFromNumeric(Typelib::Val
...
                     break;
```
Comment (jmachowinski) [inline src/TypelibConfiguration.cpp:110]: same as above

**Comment by jmachowinski** [comment]
Can you change the 2 lines and test if it compiles ?

**Comment by planthaber** [comment]
updated and it compiles, didn't test functionality

**Pull request merged**